--- debian/rules	2017-01-30 01:52:41.499268170 +0000
+++ debian/rules	2017-01-30 01:53:34.869262974 +0000
@@ -21,15 +21,17 @@
 export HOST_CFLAGS
 export HOST_LDFLAGS
 export TARGET_CPPFLAGS := -Wno-unused-but-set-variable
-export TARGET_LDFLAGS := -no-pie
+#export TARGET_LDFLAGS := -no-pie
+export TARGET_LDFLAGS :=
+export PATH := /home/vagrant/dh_new:/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin
 
-ifeq (,$(shell which qemu-system-i386 2>/dev/null))
+ifeq (,$(shell which xqemu-system-i386 2>/dev/null))
 with_check := no
 else
 with_check := yes
 endif
 
-CC := gcc-6
+CC := gcc
 
 confflags = \
 	PACKAGE_VERSION="$(deb_version)" PACKAGE_STRING="GRUB $(deb_version)" \
@@ -40,7 +42,7 @@
 
 AUTOGEN_DEB_FILES = config templates preinst postinst postrm dirs install links maintscript
 
-BUILD_PACKAGES := $(strip $(shell dh_listpackages))
+BUILD_PACKAGES := grub-common grub2-common grub-pc-bin grub-pc 
 # REAL_PACKAGES build an actual grub variant (and therefore have both configure
 # and build stages). EXTRA_PACKAGES do not build grub but may depend on a
 # REAL_PACKAGE (and therefore only have a build stage)
